### PR TITLE
added ability to copy comments on long press

### PR DESCRIPTION
**Pull request by harshlele**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

Added ability to copy comments on long pressing them, and entry in the settings to enable/disable it.

### CODE REVIEW

**Comment by yausername** [comment]
I don't think we need a settings option for this feature unless it is causing some performance hit.
We don't want to have a bloated settings menu.

**Comment by harshlele** [comment]
ok i'll remove it

**Comment by B0pol** [comment]
How it works? Is it like description, you can select the area you want, or it copy the whole description?

**Comment by harshlele** [comment]
it copies the whole comment. i actually wanted to make it so that you can select the text, but the root view of the comment is clickable, so the the textview isn't selectable.

File: app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java [hunk 1]
```diff
@@ -120,6 +126,21 @@ public void updateFromItem(final InfoItem infoItem, final HistoryRecordManager h
...
         });
+
+
+        itemView.setOnLongClickListener(new View.OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View view) {
+
+                ClipboardManager clipboardManager = (ClipboardManager) itemBuilder.getContext()
+                                                    .getSystemService(Context.CLIPBOARD_SERVICE);
+                clipboardManager.setPrimaryClip(ClipData.newPlainText(null,itemContentView.getText()));
```
Comment (yausername) [inline app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java]: `itemContentView.getText()` can sometimes copy the ellipsized text. Use `commentText` instead.
```suggestion
clipboardManager.setPrimaryClip(ClipData.newPlainText(null,commentText));
```

File: app/src/main/res/xml/content_settings.xml [hunk 2]
```diff
@@ -65,6 +65,7 @@
         android:title="@string/show_comments_title"
         android:summary="@string/show_comments_summary"/>
 
+
```
Comment (yausername) [inline app/src/main/res/xml/content_settings.xml]: remove empty line

**Pull request merged**